integrations/operator/crdgen/testdata/protofiles/teleport/accesslist/v1/accesslist.proto

```diff
@@ -0,0 +1,115 @@
+// Copyright 2023 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package teleport.accesslist.v1;
```

Comment:
You don't need to copy all these files.

Reply:
In a near future I think I'll just remove the testdata and use the actual proto files from `api/proto`, updating them is cumbersome and we end up with XXL PRs for nothing

Reply:
I just ran `make update-protos`, but I'll go ahead and delete all unnecessary files